agent/src/beerocks/slave/agent_ucc_listener.cpp

```diff
@@ -75,7 +75,7 @@ bool agent_ucc_listener::validate_destination_alid(const std::string &dest_alid)
 {
     // On the agent side, the dest_alid is not really needed since the destination socket will
     // always be the controller socket.
-    return (*m_controller_sd)->getPeerMac() == dest_alid;
+    return m_backhaul_manager_ctx.get_controller_bridge_mac() == dest_alid;
```

Comment:
This shouldn't be using the bridge mac. We should add a confiruation parameter to the agent which is al_mac, and for now set it to the same address as the bridge. Or at least call it here and everywhere we use it al_mac and not controller_bridge_mac, since in the future this might change.

Reply:
Since we work now with birdge mac, I prefer to use `bridge` terminology to prevent confusion. When we will the bridge mac will stop from being the bridge mac we will change it.

Reply:
I think this proves that we should clean up our terminology sooner rather than later :-)

Since currently indeed controller_bridge_mac is used in a few places already, I agree that it's better to use the same term in new code in backhaul_manager and son_slave_thread, even if it is wrong.

Reply:
We can at least add a comment in those places saying its the al_mac..